Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml

```
@@ -92,8 +93,8 @@
     sprintModifier: 0.25
   - type: CursorOffsetRequiresWield
   - type: EyeCursorOffset
-    maxOffset: 3
-    pvsIncrease: 0.3
+    maxOffset: 10 # Sunrise-Edit
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)